### PR TITLE
chore: App Router Migration - Fix metadata util used in App Router and remove duplicate code

### PR DESCRIPTION
**Pull request by hbjORbj**
## What does this PR do?

- Removes `WizardLayoutAppDir` and replace its instances because it's exactly the same as `WizardLayout`
- Fixes a util function for generating metadata in App Router pages: `preparePageMetadata`. It wasn't generating a description, and it was generating an incorrect title
    - Remove `| ${APP_NAME}` from metadata title (it will be added in `preparePageMetadata` util)

## Mandatory Tasks (DO NOT REMOVE)

- [x] I have self-reviewed the code (A decent size PR without self-review might be rejected).
- [ ] I have added a Docs issue [here](https://github.com/calcom/docs/issues/new) if this PR makes changes that would require a [documentation change](https://docs.cal.com/docs). If N/A, write N/A here and check the checkbox.
- [x] I confirm automated tests are in place that prove my fix is effective or that my feature works.

## How should this be tested?

- Compare the same route in App / Pages router and see the difference in metadata tab name

### CODE REVIEW

File: apps/web/app/future/apps/[slug]/[...pages]/page.tsx [hunk 1]
```diff
@@ -65,7 +64,7 @@ export const generateMetadata = async ({
 
   if (mainPage === "forms") {
     return await _generateMetadata(
-      () => `Forms | ${APP_NAME}`,
```
Comment (hbjORbj) [inline apps/web/app/future/apps/[slug]/[...pages]/page.tsx:68]: `| ${APP_NAME}` should not be added here. It should be rather added in `preparePageMetadata`.

**Review by joeauyeung — DISMISSED** [review]
Love seeing the refactors made here. I had some comments but I noticed the same changes are in #16334 so I'll comment there.

**Comment by hbjORbj** [comment]
@joeauyeung Hey Joe, thanks for the review! Now that the PR is approved, what are the next steps prior to merging this PR? Let me know what the usual workflow is! :D

**Comment by joeauyeung** [comment]
Before merging we need to add the `ready-for-e2e` label which will run the E2E tests. Once those are passing we can merge into `main`.

**Comment by hbjORbj** [comment]
@joeauyeung Seems like all e2e tests pass. I can add `auto-merge` label, right?

**Review by joeauyeung — APPROVED** [review]

**Pull request merged**